MC/config/QC/json/mft-tracks-mc.json

```diff
@@ -0,0 +1,67 @@
+{
+    "qc" : {
+      "config" : {
+        "database" : {
+          "implementation" : "CCDB",
+          "host" : "ccdb-test.cern.ch:8080",
+          "username" : "not_applicable",
+          "password" : "not_applicable",
+          "name" : "not_applicable"
+        },
+        "Activity" : {
+          "number" : "42",
+          "type" : "2",
+          "provenance": "qc_mc",
+          "passName": "passMC",
+          "periodName": "SimChallenge"
+        },
+        "monitoring" : {
+          "url" : "no-op://"
+        },
+        "consul" : {
+          "url" : ""
+        },
+        "conditionDB" : {
+          "url" : "alice-ccdb.cern.ch"
+        }
+      },
+      "tasks" : {
+        "TracksMC" : {
+          "active" : "true",
+          "className" : "o2::quality_control_modules::mft::QcMFTTrackMCTask",
+          "moduleName" : "QcMFT",
+          "detectorName" : "MFT",
+          "cycleDurationSeconds" : "30",
+          "maxNumberCycles" : "-1",
+          "dataSource_comment" : "The other type of dataSource is \"direct\", see basic-no-sampling.json.",
+          "dataSource" : {
```

Comment:
If your sampling fraction is 100%, please use `direct` so that we can avoid using the dispatcher.

Reply:
Hi Jian, thanks, I have changed the dataSource type to `direct`.